docs/photutils/getting_started.rst

```diff
@@ -21,9 +21,10 @@ background-subtracted.
 Photutils supports several source detection algorithms.  For this
 example, we use :func:`~photutils.daofind` to detect the stars in the
 image.  We set the detection threshold at the 3-sigma noise level,
-estimated using the median absolution deviation of the image.  The
-parameters of the detected sources are returned as an Astropy
-`~astropy.table.Table`:
+estimated using the `median absolute deviation <http://docs.astropy.org/en/
+stable/api/astropy.stats.mad_std.html#astropy.stats.mad_std>`_ of the image.
```

Comment:
I'm a bit uncomfortable with hard wired links like this. We use the astropy intersphinx, so it would be better to stick to it whenever possible (thus we will be insensitive of any structural changes of this function within astropy).
However I'm not sure how to add custom text to an intersphinx link, so maybe you can say here something along the line of: ``estimated using the median absolution deviation (using `~astropy.stats.mad_std`) of the image.``